### PR TITLE
[FIX] website: fix multiple buttons rendering within snippets

### PR DESCRIPTION
**Pull request by chgo-odoo**
This commit introduces a fix for the snippets that include at least two
buttons in their layout.

Prior to this commit, the buttons were simply defined within the `<p>`
tag, due to a lack of testing corner case scenarios. This implementation
was actually wrong, because as soon as you add `n+1` button within the
editor, it'll apply a `mb-2` on each button to space them correctly on
smaller devices.

To fix this issue, we simply replicate the behaviour of the editor by
adding the class on the snippets that match this scenario.

task-4210852
part of task-4077427

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website/views/snippets/s_discovery.xml [hunk 1]
```diff
@@ -4,15 +4,19 @@
...
-                <i class="fa fa-fw fa-info-circle o_not-animable" role="img"/>&#160;&#160;What do you want to promote&amp;nbsp;? &#160;&#160;&#160;&#160;<a href="#">See more <i class="fa fa-long-arrow-right" role="img"/></a>
-            </span>
-            <h1 class="display-2" style="text-align: center;">Discover our solutions</h1>
-            <p class="lead" style="text-align: center;">Write one or two paragraphs describing your product, services or a specific feature.<br/> To be successful your content needs to be useful to your readers.<br/><br/></p>
-            <p style="text-align: center;">
-                <a t-att-href="cta_btn_href" class="btn btn-primary"><t t-out="cta_btn_text">Our store</t></a>
-                <a t-att-href="cta_btn_href" class="btn btn-secondary"><t t-out="cta_btn_text">Contact us</t></a>
-            </p>
+            <div class="row s_nb_column_fixed">
+                <div class="col-12 col-lg-8 offset-lg-2">
```
Comment (qsm-odoo) [inline addons/website/views/snippets/s_discovery.xml:8]: What's the point of this extra row and fixed col if everything is centered anyway? Is that related to this commit's work about buttons?

**Review by qsm-odoo — CHANGES_REQUESTED** [review]

File: addons/website/views/snippets/s_discovery.xml (hunk 1 above)
Comment (chgo-odoo) [reply addons/website/views/snippets/s_discovery.xml:8]: You're right, this diff shouldn't be part of this commit, it's related to [this PR](https://github.com/odoo/odoo/pull/181509), where we need to define a col to prevent the text for taking the whole container size, and even within that PR, it should be widespread to some other snippets, I'll remove it from this commit

**Review by qsm-odoo — APPROVED** [review]
@robodoo r+
